_posts/2020-05-27-cockpit-220.md

```diff
@@ -0,0 +1,45 @@
+---
+title: Cockpit 220
+author: marusak
+date: 2020-05-27
+tags: cockpit linux
+slug: cockpit-220
+category: release
+summary: Cockpit with new navigation
+comments: 'true'
+---
+
+Cockpit is the [modern Linux admin interface](https://cockpit-project.org/). We release regularly. Here are the release notes from version 220.
+
+### New navigation with integrated hosts switching
+
+The navigation has been redesigned and brings four major improvements:
+- **One level navigation**: The current two level navigation has been squashed for better use of space and better discoverability
+- **Integrated hosts switching**: Switching between hosts as well as editing them can be done now directly from the navigation without using the 'Dashboard' component
+- **Better discoverability of applications**: Applications are shown as the first group in the menu and are also searchable
+- **Super indicator stuff**: Yeah, lets mention #14055 here
+
+![New one level navigation](/images/one-level-nav.png)
```

Comment:
Screenshot is too big IMHO, and scaled down. How about cropping out just the nav menu and a little bit of the overview page, and then putting the host switcher one side by side?

Reply:
The top right corner changed as well, I want that in screenshot as well. But I can make it bigger and the menu more compact to have better screenshots

Reply:
Ah, fair enough -- I usually make my browser around 1000px wide for screenshots (i. e. as narrow as possible to still see everything decently, but avoiding useless whitespace), not the full 1920.